Articles/ShortArticles/TheDeveloperCertificateOfOrigin.md

```diff
@@ -0,0 +1,147 @@
+# The Developer Certificate of Origin
+
+<!--deck text start-->
+Every project that accepts contributions from outside developers should have a legal mechanism in place that allows those contributions to become part of the main code base that allows for further development and distribution and avoids future legal problems.
+The Developer Certificate of Origin with signoffs embedded into Git commits messages is becoming the preferred way to handle this issue and is being adopted by important open source projects.
+<!--deck text end-->
+
+#### Contributed by [Roscoe A. Bartlett](https://github.com/bartlettroscoe)
+
+#### Publication date: ???, 2021
+
+When a person writes a piece of software, by default, they own that software unless some other legal entity -- such as their employer -- owns their work by default.
```

Comment:
I propose changing the first paragraph to read as follows:

When a person writes a piece of software, by default, they own the copyrights in that software by default, unless it is a "work for hire", in which case their employer owns it, or those rights are coveyed to some other legal entity by a written agreement.
And even if that person provides that source code with the implied intent to make it part of some existing software project or otherwise give it away (i.e. place it in the public domain), legal experts may differ as to whether that by itself is enough to allow the receiving project to modify and distribute the contribution along with the rest of the project.  For some experts, contributions are implictly offered under the same licensing terms as the code base to which the contribution is being made.  Others prefer to have the owner of the copyright in new contributions 
explicitly state that they are providing the software (or a modification to an existing piece of software) to a project in accordance to some specific software license associated with the project<sup>[0]</sup>.
It is the process by which a person provides an (updated) contribution to an existing project that has an existing software license that is the focus of this article.

Reply:
Should be addressed in new commit badd685f8f5bf9f11a7a345e495083b25ecd127e